### DISCUSSION TITLE
Add a DUMMY trigger/button to help debug Flows

### DISCUSSION THREAD

**Discussion by erondpowell** [directus/directus#15891] [Feature Requests]
Currently, you either have to flip from `content` module back to `config>flows` again and again (for event hooks and triggers).

To work around this, I have been using Postman to trigger flows as much as possible. 

I'm wondering if we could add a button/interface *inside the flow dashboard-grid aread itself* that lets us trigger the flow with Dummy Data. This would be useful for all triggers:

- Event Hooks
- Another Flow
- Webooks
- Manual Button
- Cron Jobs

To do this we'd probably need a form input, to specify the dummy `$trigger` data for the flow.

*(Note: Although cron jobs take no data and are.... well... "triggered by time"... it might be nice to just have a button that triggers the flow once by default, but also lets you input a time in the form input- so it isn't constantly triggering during debugging. Also note my other discussion on Cron Job Triggers: https://github.com/directus/directus/discussions/15652)*

**Comment by m-kind** [comment]
As an additional feature (and maybe dead simple to implement) I'd love to have a "re-run flow" button next to each of the flow execution log entries. The original payload is already there and this could make life much easier e.g. in case a flow that communicates with an external system fails (because of a problem in that system - not in the flow itself). This would be quite similar to the "re-run job" feature in CI/CD pipelines (GitLab, Bitbucket, ...)

**Comment by vanling** [comment]
Good discussion:) 

For CRON. I just made one haha. Had it on * * * * * every minute, waiting a minute to see if it worked or not. A quick "trigger" would be very very very nice there. 

💡  _Maybe from the logs on the side have an option to retry/replay the trigger with same data?, This would help with the update/create event hooks_

**Comment by azrikahar** [comment]
There's also an existing discussion on this, particularly the form input, over here: https://github.com/directus/directus/discussions/14010#discussioncomment-3002473 👍

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
